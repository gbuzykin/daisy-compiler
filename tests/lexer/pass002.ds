const i8_max = 0x7fi8; const u8_max = 0xffu8;
const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
const auto_max = 0xffffffffffffffff;
