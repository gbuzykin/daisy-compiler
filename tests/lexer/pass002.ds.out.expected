./lexer/pass002.ds:1:1: debug: token
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   | ^~~~~
./lexer/pass002.ds:1:7: debug: id: i8_max
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |       ^~~~~~
./lexer/pass002.ds:1:14: debug: token
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |              ^
./lexer/pass002.ds:1:16: debug: integer number: 127
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |                ^~~~~~
./lexer/pass002.ds:1:22: debug: token
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |                      ^
./lexer/pass002.ds:1:24: debug: token
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |                        ^~~~~
./lexer/pass002.ds:1:30: debug: id: u8_max
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |                              ^~~~~~
./lexer/pass002.ds:1:37: debug: token
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |                                     ^
./lexer/pass002.ds:1:39: debug: integer number: 255
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |                                       ^~~~~~
./lexer/pass002.ds:1:45: debug: token
 1 | const i8_max = 0x7fi8; const u8_max = 0xffu8;
   |                                             ^
./lexer/pass002.ds:2:1: debug: token
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   | ^~~~~
./lexer/pass002.ds:2:7: debug: id: i16_max
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |       ^~~~~~~
./lexer/pass002.ds:2:15: debug: token
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |               ^
./lexer/pass002.ds:2:17: debug: integer number: 32767
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |                 ^~~~~~~~~
./lexer/pass002.ds:2:26: debug: token
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |                          ^
./lexer/pass002.ds:2:28: debug: token
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |                            ^~~~~
./lexer/pass002.ds:2:34: debug: id: u16_max
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |                                  ^~~~~~~
./lexer/pass002.ds:2:42: debug: token
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |                                          ^
./lexer/pass002.ds:2:44: debug: integer number: 65535
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |                                            ^~~~~~~~~
./lexer/pass002.ds:2:53: debug: token
 2 | const i16_max = 0x7fffi16; const u16_max = 0xffffu16;
   |                                                     ^
./lexer/pass002.ds:3:1: debug: token
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   | ^~~~~
./lexer/pass002.ds:3:7: debug: id: i32_max
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |       ^~~~~~~
./lexer/pass002.ds:3:15: debug: token
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |               ^
./lexer/pass002.ds:3:17: debug: integer number: 2147483647
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |                 ^~~~~~~~~~~~~
./lexer/pass002.ds:3:30: debug: token
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |                              ^
./lexer/pass002.ds:3:32: debug: token
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |                                ^~~~~
./lexer/pass002.ds:3:38: debug: id: u32_max
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |                                      ^~~~~~~
./lexer/pass002.ds:3:46: debug: token
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |                                              ^
./lexer/pass002.ds:3:48: debug: integer number: 4294967295
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |                                                ^~~~~~~~~~~~~
./lexer/pass002.ds:3:61: debug: token
 3 | const i32_max = 0x7fffffffi32; const u32_max = 0xffffffffu32;
   |                                                             ^
./lexer/pass002.ds:4:1: debug: token
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   | ^~~~~
./lexer/pass002.ds:4:7: debug: id: i64_max
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |       ^~~~~~~
./lexer/pass002.ds:4:15: debug: token
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |               ^
./lexer/pass002.ds:4:17: debug: integer number: 9223372036854775807
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |                 ^~~~~~~~~~~~~~~~~~~~~
./lexer/pass002.ds:4:38: debug: token
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |                                      ^
./lexer/pass002.ds:4:40: debug: token
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |                                        ^~~~~
./lexer/pass002.ds:4:46: debug: id: u64_max
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |                                              ^~~~~~~
./lexer/pass002.ds:4:54: debug: token
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |                                                      ^
./lexer/pass002.ds:4:56: debug: integer number: 18446744073709551615
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |                                                        ^~~~~~~~~~~~~~~~~~~~~
./lexer/pass002.ds:4:77: debug: token
 4 | const i64_max = 0x7fffffffffffffffi64; const u64_max = 0xffffffffffffffffu64;
   |                                                                             ^
./lexer/pass002.ds:5:1: debug: token
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   | ^~~~~
./lexer/pass002.ds:5:7: debug: id: i_max
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |       ^~~~~
./lexer/pass002.ds:5:13: debug: token
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |             ^
./lexer/pass002.ds:5:15: debug: integer number: 9223372036854775807
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |               ^~~~~~~~~~~~~~~~~~~
./lexer/pass002.ds:5:34: debug: token
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |                                  ^
./lexer/pass002.ds:5:36: debug: token
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |                                    ^~~~~
./lexer/pass002.ds:5:42: debug: id: u2_max
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |                                          ^~~~~~
./lexer/pass002.ds:5:49: debug: token
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |                                                 ^
./lexer/pass002.ds:5:51: debug: integer number: 18446744073709551615
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |                                                   ^~~~~~~~~~~~~~~~~~~
./lexer/pass002.ds:5:70: debug: token
 5 | const i_max = 0x7fffffffffffffffi; const u2_max = 0xffffffffffffffffu;
   |                                                                      ^
./lexer/pass002.ds:6:1: debug: token
 6 | const auto_max = 0xffffffffffffffff;
   | ^~~~~
./lexer/pass002.ds:6:7: debug: id: auto_max
 6 | const auto_max = 0xffffffffffffffff;
   |       ^~~~~~~~
./lexer/pass002.ds:6:16: debug: token
 6 | const auto_max = 0xffffffffffffffff;
   |                ^
./lexer/pass002.ds:6:18: debug: integer number: 18446744073709551615
 6 | const auto_max = 0xffffffffffffffff;
   |                  ^~~~~~~~~~~~~~~~~~
./lexer/pass002.ds:6:36: debug: token
 6 | const auto_max = 0xffffffffffffffff;
   |                                    ^
./lexer/pass002.ds: info: warnings 0, errors 0
